test/system/030-run.bats

```diff
@@ -589,4 +589,22 @@ json-file | f
     is "${lines[1]}" "$rand" "Container runs successfully despite warning"
 }
 
+# Regression test for #9040
+@test "podman run - implicitly create workdir" {
+    workdir=/$(random_string 10)
+    PODMAN_TIMEOUT=240 run_podman build -t workdir_test - << EOF
+FROM  $IMAGE
+WORKDIR $workdir
+EOF
```

Comment:
What would you think of this instead?
```diff
diff --git a/test/system/070-build.bats b/test/system/070-build.bats
index 0e83a184b..c81918428 100644
--- a/test/system/070-build.bats
+++ b/test/system/070-build.bats
@@ -148,7 +148,6 @@ EOF
     cat >$tmpdir/Containerfile <<EOF
 FROM $IMAGE
 LABEL $label_name=$label_value
-RUN mkdir $workdir
 WORKDIR $workdir

 # Test for #7094 - chowning of invalid symlinks
```

Reply:
I copied the test from there and thought about it but thought you'd prefer to have it in `030-run` :smile:   I can certainly change the build test but keep the `--workdir=...` here :+1:

Reply:
I think the tests are failing because the working dir is on a volume, so the check fails to detect it